doc-surrealdb_versioned_docs/version-latest/surrealql/functions/database/index.mdx

```diff
@@ -15,96 +15,119 @@ SurrealDB comes with a large number of in-built functions for checking, manipula
   <thead>
     <tr>
       <th scope="col">Module</th>
-      <th scope="col">Example / Notes</th>
+      <th scope="col">Description</th>
```

Comment:
suggestion: maybe we can use `colspan` for the description to more space than the rest of the columns.

Reply:
We can adjust the width of each column this way, but I think many of the descriptions and examples will span multiple lines and appear somewhat cramped either way. This may appear especially awkward for the examples since they short snippets of code intended to span one line.

Reply:
Well, I suggested this because `colspan` is already used for other tables. By removing the suffix, the first column would not need enough space where the description currently take multiple lines.

Reply:
I tried setting a precise column width ratio that fits the content of each column as best as I could, using the `style` attribute.

```html
<th scope="col" style="width:15%">Function Module</th>
<th scope="col" style="width:50%">Description</th>
<th scope="col" style="width:35%">Example</th>
```

Let me know if this looks alright, or if we want to consider omitting the "Examples" column to make more room for the "Description" column.

The GitHub preview of the page will not display the proportional column widths, but they can be seen if the page is rebuilt. I have included a screenshot below for convenience.

![image](https://github.com/user-attachments/assets/cf75bad7-0685-4451-a94e-fa034d2e81b4)

Reply:
I suppose so. Having the example in two lines feels unattractive. Do we really need to set an example in this page @Ekwuno?

If we do, I would suggest to remove the column and put that below the description, using a [CodeBlock](https://docusaurus.io/docs/markdown-features/code-blocks#usage-in-jsx) with a `title` and the `language="surql"`.

Reply:
I originally went with a single example for each in a recent PR that adds the table to the top of the page because otherwise there isn't much for the eye to catch (especially in the case of an English L2 user). With a single example for each you can see that `crypto` is the place to hash a password, HTTP lets you use `GET`, and so on - without having to click to explore a second page to confirm.

They are meant to be as short as possible though so further shrinking would look good. math::max can be shrunk, the "MyPaSSw0RD" can be changed to "1234" or something, etc.

Reply:
@Dhghomon Thanks for the added perspective and the strong argument; I had not considered how especially useful the examples are to English L2 users.

To add another argument for the examples' value: Surreal DB has excellent function syntax. The decision to have the ubiquitous format `module::function` makes the intent and result of code very obvious yet concise. This distinguishes SurrealDB from many other database solutions. As an up-and-coming database looking aiming to grow its user base, I think the examples serve as a great showcase of this benefit, and will do a lot of work to impress readers.

@Ekwuno @Dhghomon @Odonno What would everyone think of the formatting in the example below, similar to what @Odonno recommended above, with each function module having its description and example in the same table cell,  with the example starting on a new line beneath the description? That way both the description and example can make use of plenty of width.

![image](https://github.com/user-attachments/assets/450f9b4a-078d-4026-a191-3422c716c6c0)

Reply:
@matthewjnield That looks pretty nice! It even makes the function module names stand out more thanks to the extra height.

Reply:
Indeed, it looks great.

Reply:
Thanks for working on this @matthewjnield looks great! I just approved the CI, and you may have some broken links because of this large PR I just merged in: https://github.com/surrealdb/docs.surrealdb.com/pull/737

If you do a search and replace to change `/docs/surrealdb/surrealql/` to `/docs/surrealql/` it should be sorted If you don't mind I can fix it but unsure if you have local changes you haven't pushed yet.

Reply:
@Ekwuno Thanks for the heads up; I will fix the broken links. If the formatting of the table is fine, then there is just the following matter left to resolve before merging this PR and closing the related issue:

"The value module does not have a description on the dedicated page, and I am not sure what to add as one. Please advise."

Any proposal for this? I can add it to both the page for the `value` module and to the function module table.

Reply:
Something like `These functions can be used when working with any value type inside SurrealDB.`.